src/config/consts.js

```diff
@@ -15,6 +15,11 @@ const POST_TYPE = {
   PUBLIC: "PUBLIC",
   PRIVATE: "PRIVATE"
 }
+const NEGOTIATION_STATUS = {
```

Comment:
add a blank line before

Reply:
fixed in 6440b8bb1142bd89287e763fa6ad151d9d64664d